jwst/datamodels/dqflags.py

```diff
@@ -16,9 +16,11 @@
 which provides 32 bits. Bits of an integer are most easily referred to using
 the formula `2**bit_number` where `bit_number` is the 0-index bit of interest.
 """
-from astropy.nddata.bitmask import interpret_bit_flags as ap_interpret_bit_flags
 
-from jwst.lib.basic_utils import multiple_replace
+# These imports are here for backwards compatibility
+from astropy.nddata.bitmask import interpret_bit_flags as ap_interpret_bit_flags # NOQA
```

Comment:
What does "NOQA" stand for in these lines?

Reply:
Flake8 complains that these are imported but not used. 
A comment `#NOQA` signals flake8 to exclude this line from the check.

Reply:
Ah, I didn't know that little trick. And it also doesn't complain about the fact that there aren't 2 spaces before the "#"?

Reply:
"No Quality Assurance" I believe.  👍